### PR TITLE
[Draft][onert-micro] Introduce training last FC layer

### PR DESCRIPTION
**Pull request by BalyshevArtem**
This draft introduces changes for training last fc lasyer

ONE-DCO-1.0-Signed-off-by: Artem Balyshev <a.balyshev@samsung.com>

### CODE REVIEW

File: onert-micro/luci-interpreter/include/luci_interpreter/TrainingSettings.h [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  EnableTrainModeError,
+  DoubleTrainModeError
+};
+
+enum MetricsTypeEnum
+{
+  Accuracy
+};
+
+enum ErrorsTypeEnum
```
[the remark this answers is not in the document]
Comment (BalyshevArtem) [reply onert-micro/luci-interpreter/include/luci_interpreter/TrainingSettings.h]: Done

File: onert-micro/eval-driver/TrainingDriver.cpp [hunk 2]
```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
```
Comment (jyoungyun) [inline onert-micro/eval-driver/TrainingDriver.cpp:2]:
```suggestion
 * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
```

**Comment by jyoungyun** [comment]
Could you share your test result? :)

**Comment by BalyshevArtem** [comment]
Merged, close it

**Pull request closed without merging**